coalib/processes/Processing.py

```diff
@@ -365,8 +370,18 @@ def instantiate_processes(section,
     # only on changed files or not. Global bears require all the files
     complete_filename_list = filename_list
 
+    # Checks if this section uses raw files or not
+    bears = chain(local_bear_list, global_bear_list)
+    use_raw_files = set(bear.USE_RAW_FILES for bear in bears)
+    if len(use_raw_files) > 1:
+        log_printer.err("Bears that uses raw files can't be mixed with "
+                        'Bears that uses text files. Skipping this '
+                        'section.')
```

Comment:
That introduces lots of edge cases, and so far we have no sensible use case for that.  All raw bears we have in the pipeline will be loading files that cant be loaded as text file.